packages/app/src/utils/getFilteredThresholdValues.ts

```diff
@@ -0,0 +1,15 @@
+import { ChoroplethThresholdsValue } from '@corona-dashboard/common';
+
+// Get all the thresholds and filter out the right color that needs to be rendered in the rectangle,
+// by filtering out all the higher thresholds and then using the last key that still exists.
```

Comment:
We ususally write comment blocks with the /** */ syntax
```suggestion
/**
 * Get all the thresholds and filter out the right color that needs to be rendered in the rectangle,
 * by filtering out all the higher thresholds and then using the last key that still exists.
 */
```